src/coreclr/scripts/superpmi.py

```diff
@@ -2723,13 +2952,13 @@ def process_base_jit_path_arg(coreclr_args):
 
         if coreclr_args.base_git_hash is None:
             # We've got the current hash; figure out the baseline hash.
-            command = [ "git", "merge-base", current_hash, "main" ]
+            command = [ "git", "branch", "-r", "--sort=-committerdate", "-v", "--list", "*/main" ]
```

Comment:
The old method does not work for my workflow, I never have a 'main' branch.
The new method is to use the newest commit of any branch that matches "*/main".

Reply:
Ah whoops, this still needs a `merge-base` call, will fix...